### PR TITLE
[TACHYON-403] Fix exception in TFsShell

### PR DESCRIPTION
**Pull request by carsonwang**
https://tachyon.atlassian.net/browse/TACHYON-403

Fix exception when tail an empty file.
Remove the <path> parameter in getUsedBytes, getCapacityBytes.
Add error message when operation mv is failed.

### CODE REVIEW

File: core/src/main/java/tachyon/command/TFsShell.java [hunk 1]
```diff
@@ -773,9 +776,7 @@ public int touch(String[] argv) throws IOException {
     }
     TachyonURI path = new TachyonURI(argv[1]);
     TachyonFS tachyonClient = createFS(path);
-    TachyonFile tFile = tachyonClient.getFile(tachyonClient.createFile(path));
-    OutputStream out = tFile.getOutStream(WriteType.THROUGH);
-    out.close();
+    tachyonClient.createFile(path);
```
Comment (calvinjia) [inline core/src/main/java/tachyon/command/TFsShell.java]: Won't this leave the file incomplete?

**Comment by haoyuan** [comment]
Jenkins, test this please.

File: core/src/main/java/tachyon/command/TFsShell.java (hunk 1 above)
Comment (carsonwang) [inline core/src/main/java/tachyon/command/TFsShell.java]: Oh, yes. Sorry for that. I'll fix it.

**Comment by haoyuan** [comment]
Jenkins, test this please.

**Comment by haoyuan** [comment]
@carsonwang  Could you please create a JIRa for this? 

http://tachyon-project.org/master/Contributing-to-Tachyon.html

**Comment by carsonwang** [comment]
Sure. a Jira TACHYON-403 is created.

**Comment by haoyuan** [comment]
Jenkins, test this please.

**Comment by calvinjia** [comment]
Jenkins, retest this please.

**Comment by calvinjia** [comment]
+1 LGTM

**Pull request merged**